server/public/index.html

```diff
@@ -4,11 +4,16 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Final Project</title>
+
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
```

Comment:
The latest version is `4.5.something`.

https://getbootstrap.com/docs/4.5/getting-started/introduction/